modules/client-configuration/pages/configuration-management.adoc

```diff
@@ -1,88 +1,90 @@
-// From reference guide 
-== Configuration Management for Salt
+[[configuration-management]]
+= Configuration Management
 
-This matrix shows supported and unsupported configuration management features.
+You can use configuration files and channels to manage configuration for your clients, rather than configuring each client manually.
 
-[IMPORTANT]
-====
-Several {webui} tabs will be missing for Salt Configuration Management.
-These features are not suitable for Salt clients.
-====
+Configuration parameters are scripted and stored in configuration files.
+You can write configuration files directly using the {productname} {webui}, or you can upload or link to files that exist in other locations.
 
-.Salt Configuration Management
-[cols="1,1", options="header"]
-|===
-| Configuration Management Features | Salt Support Status
-| Global Configuration Channels     | Supported
-| Deploying Files | Supported
-| Comparing Files | Supported (but the logic is currently inverted)
-| Locally Managed Files | Unsupported
-| Sandbox Files | Unsupported
-| Applying the Highstate | Apply the highstate and configuration channels will be deployed to all subscribed systems.
-| File Import from a Client | Unsupported
-| Configuration Macros | Unsupported
-|===
+Configuration files can be centrally managed, or locally managed.
+Centrally managed configuration files are provided by global configuration channels and can be applied to any client subscribed to the {productname} Server.
+Locally managed configuration files are used to override centrally managed configuration settings.
+They are especially useful for {productname} users who do not have configuration administration privileges, but need to make changes to the clients that they manage.
 
```

Comment:
I think this kind of table is needed and would be helpful as we don't support all the features for salt clients.

Reply:
So you're happy with the content as it stands, then?

Reply:
Ok I guess I wasn't clear, sorry. What I am asking is that we need this [change](https://github.com/uyuni-project/uyuni-docs/pull/409/files#diff-70bea3b51fd07496cfd572df7127c8d4L6-L24) back  This was good information to indicate what is not supported in salt clients which in this PR has been removed.

Reply:
heh, well, you know how much i love a support matrix! :woman_facepalming:

Reply:
![image](https://user-images.githubusercontent.com/3914967/90837523-a10f7b00-e395-11ea-8446-ba9b500bd6df.png)

Added, and updated syntax and format to match the others.